force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,7 +132,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+        String result = '';
+
+        if (a == '') {
+            result = 'Empty';
+        } else if (a == null) {
+            result = 'Null';
+        } else {
+            result = 'Contains Text';
+        }
+
+        return result; // Replace null with the variable you used to store the result
     }
```

Comment:
Good work. My solution is below as an example of an alternative and the style I like to use. 

```
public static String checkString(String a) {
        if (a == null) {
            return 'Null';
        }
        if (String.isEmpty(a)) {
            return 'Empty';
        }
        return 'Contains Text';
    }
    ```